pytwis/pytwis_clt.py

```diff
@@ -223,20 +230,20 @@ def pytwis_command_parser(raw_command):
         arg_dict = parse.parse('{username} {password}', args)
```

Comment:
Could you please replace '{username} {password}' by the ARG constants?

```python
arg_dict = parse.parse('{{{arg1}}} {{{arg2}}}'.format(arg1=ARG_USERNAME, arg2=ARG_PASSWORD), args)
```

Reply:
Fixed